README.md

```diff
@@ -36,6 +36,72 @@ Deploy and verify a contract
 source .env
 
 # To deploy and verify our contract
-forge script script/WikiNoValidator.s.sol:WikiNoValidator --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast --verify --etherscan-api-key $ETHERSCAN_KEY -vvvv --gas-price 60 --legacy
+forge script script/BrainPassDeployer.s.sol:BrainPassDeployer --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast --verify --etherscan-api-key $ETHERSCAN_KEY -vvvv --gas-price 60 --legacy
 
 ```
+
+# Expalaining How BrainPassCollectibles Works
+
+BrainPassCollectibles is a Solidity smart contract that enables the creation and management of BrainPass NFTs. It allows users to buy passes to have some Wiki previledges on IQ Wiki and provides functionalities to mint NFTs, increase pass time, and manage pass types.
+
+## Usage
+
+### Contract Deployment
+
+Deploy the `BrainPassCollectibles` contract by providing the address of the IQ token contract as a constructor parameter.
+
+### Adding a Pass Type
+This function allows the contract owner to add a new pass type.
+
+Call the `addPassType` function with the following parameters:
```

Comment:
shouldnt have  a delPassType / pauseMintPassType in case you create a wrong pass type? 

for example, if you by mistake create a passtype that price is 0, contract wouldnt have any way to fix that

Reply:
okay @kesar I think a pauseMintPass func will do that